base/lib/core/presenter/CoreApplicationPlugin.cpp

```diff
@@ -85,23 +86,8 @@ void CoreApplicationPlugin::help()
 
 void CoreApplicationPlugin::about()
 {
-  auto version_text = QStringLiteral("%1.%2.%3-%4 '%5'\n\n")
-                          .arg(SCORE_VERSION_MAJOR)
-                          .arg(SCORE_VERSION_MINOR)
-                          .arg(SCORE_VERSION_PATCH)
-                          .arg(SCORE_VERSION_EXTRA)
-                          .arg(SCORE_CODENAME);
-
-  QString commit{GIT_COMMIT};
-  if (!commit.isEmpty())
-  {
-    version_text += tr("Commit: \n%1\n").arg(commit);
-  }
-
-  QMessageBox::about(
-      nullptr, tr("About score"),
-      tr("With love and sweat from the score team. \nVersion:\n")
-          + version_text);
+  AboutDialog aboutDialog;
```

Comment:
so, there's sometimes a bug with dialogs in Qt that has led to crashes in my experience ; to be safe from it instead do : 

    auto aboutDialog = new AboutDialog;
    aboutDialog->exec();
    aboutDialog->deleteLater();